File: server/response_header.py
class ResponseHeader:
  def __init__(self, version, code, payload_size):
    self.version = version
    self.code = code
    self.paylad_size = payload_size

  def get_version(self):
    return self.version
  
  def get_code(self):
    return self.code
  
  def get_payload_size(self):
    return self.paylad_size

File: client/response_header.cpp
#include "response_header.h"

ResponseHeader::ResponseHeader(uint8_t version, uint16_t code, uint32_t payload_size) 
  : version(version)
  , code(code)
  , payload_size(payload_size) {}

uint8_t ResponseHeader::get_version() {
  return this->version;
}

uint16_t ResponseHeader::get_code() {
  return this->code;
}

uint32_t ResponseHeader::get_payload_size() {
  return this->payload_size;
}